source-google-analytics-data-api/tests/snapshots/snapshots__spec__capture.stdout.json

```diff
@@ -137,7 +139,7 @@
     },
     "documentationUrl": "https://go.estuary.dev/RIcwJU",
     "oauth2": {
-      "accessTokenBody": "{\"grant_type\": \"authorization_code\", \"client_id\": \"{{{ client_id }}}\", \"client_secret\": \"{{{ client_secret }}}\", \"redirect_uri\": \"{{{ redirect_uri }}}\", \"code\": \"{{{ code }}}\"}",
+      "accessTokenBody": "grant_type=authorization_code&client_id={{#urlencode}}{{{ client_id }}}{{/urlencode}}&client_secret={{#urlencode}}{{{ client_secret }}}{{/urlencode}}&redirect_uri={{#urlencode}}{{{ redirect_uri }}}{{/urlencode}}&code={{#urlencode}}{{{ code }}}{{/urlencode}}",
```

Comment:
I'm not sure about this change to the oauth2 `accessTokenBody`. It might be fine, but we need to verify that oauth2 still works with the new connector.

Reply:
@williamhbaker Sure, do you want me to test that on my dashboards.estuary local version?

Reply:
Yeah I think that would probably be the easiest way.

Reply:
Can you remind me how to update the connectors list? I remember when accessing it the first time and only receiveing the postgres images on my local tilt dashboards.estuary

Reply:
There's a script for adding a different connector to the localstack here that you can use: https://github.com/estuary/flow/blob/master/scripts/install-connector-local.sh

Thinking about this and better understanding your original question though, testing oauth probably won't work with a local stack since the callback URL won't be correct, perhaps among other things. But give it a try and see how it goes. If that doesn't work we'll have to come up with something else. This is a pretty annoying thing to have to test but we have broken oauth before on different connectors so I think its good to do.

Reply:
To test the oauth ( as effectivly as i can), i setup a local flow dashboards and updated all variables to get to the last step ( sending the request payload via functions/oauth/v1 )

Heres the payload:
 
![Screenshot from 2024-04-04 16-51-00](https://github.com/estuary/connectors/assets/14100959/40a05d97-f768-495a-bbd4-3f45076f45f8)


As you can see, it is exactly the same as of the other google sources we have, only the scope being different

Reply:
Can you compare the payload with the current ATF version of the connector to make sure its the same?

Reply:
Thats a great ideia, thanks
will update it here soon

Reply:
@williamhbaker The payload was indeed not the same! just made the changes here 8268feab622a04c058843224e08626422616cf9f

Also updated the redirect_uri and finally got a green light on OAuth2:
![Screenshot from 2024-04-04 17-23-53](https://github.com/estuary/connectors/assets/14100959/8ba6d751-d7e3-43bb-9aab-1efdda00fef6)